common/app/views/fragments/nav/userAccountLinks.scala.html

```diff
@@ -14,49 +13,47 @@
     </li>
 
     <li class="menu-item u-h js-navigation-account-details">
-        <details>
-            <summary class="menu-item__title"
-                     data-link-name="nav2 : my account">
-                <i class="menu-item__toggle"></i>
-                my account
-            </summary>
+        <button class="menu-item__title js-navigation-toggle"
```

Comment:
@NataliaLKB Do you know whether we have the user-id at this point, which we could then use to build the link to the "my account" page?